### PR TITLE
Content endpoints

### PR DESCRIPTION
**Pull request by 66eli77**
## Summary

Use django_rest_framework to expose content app APIs to the front-end
All endpoints are directly map to the APIs.

## TODO

- [ ] Have tests been written for the new code?
- [x] Has documentation been written/updated?
- [x] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance

you can run the tests with <kolibri manage test -- kolibri.content>

## Documentation

https://docs.google.com/document/d/1XXXNA2ae8hHt3p7JuiGFDtHYPckgdVEkPdG01bGOWtE/edit#heading=h.lo7meftazil4

### CODE REVIEW

File: kolibri/content/serializers.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from models import ContentMetadata, File, Format
+from rest_framework import serializers
+
+
+class ContentMetadataSerializer(serializers.ModelSerializer):
```
Comment (rtibbles) [inline kolibri/content/serializers.py:35]: How do we set the channel with this?

One appealing solution is the namespace the content metadata serializer by channel, so the endpoint url looks something like this:

`api/<channel_id>/content/<content_id>`

So all content endpoints could be namespaced by channel in the URL?

Comment (66eli77) [inline kolibri/content/serializers.py:35]: I don't think serializer is database specific, it only needs to know the schema in order to properly serialize the queryset.

File: kolibri/content/urls.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
```
Comment (rtibbles) [inline kolibri/content/urls.py:1]: The URLs and the resources defined here are not RESTful.

You should not use GET parameters to specify the resource id.

See above for my suggestion on how we should namespace the urls for content endpoints by channel_id.

File: kolibri/content/serializers.py (hunk 1 above)
Comment (66eli77) [inline kolibri/content/serializers.py:35]: Oh no, serializer is indeed database specific, it has a save() function that will operate on models

Comment (66eli77) [inline kolibri/content/serializers.py]: @rtibbles  do you have any ideas how to dynamically namespace?

File: kolibri/content/urls.py (hunk 2 above)
Comment (66eli77) [inline kolibri/content/urls.py:1]: @rtibbles do you have any ideas how to dynamically namespace the urls?

**Comment by 66eli77** [comment]
@rtibbles test passed, conflicts resolved, let's get it merged

**Pull request merged**